controllers/ManagementController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace app\controllers;
+
+use app\controllers\mainController\MainController;
+use app\models\databaseModels\User;
+use Yii;
+use yii\data\ActiveDataProvider;
+use app\models\ModifyAccountForm;
+
+
+class ManagementController extends MainController
+{
+    /**
+     * Display a page where the accounts are shown
+     */
+    public function actionAccounts($id = null)
```

Comment:
The method name should be more descriptive

Reply:
But it's more logical to call for "accounts" in the URL (www.example-site.com/accounts), than calling for "showAccounts" (www.example-site.com/show-accounts).